.circleci/bazel.rc

```diff
@@ -28,4 +28,4 @@ build --experimental_repository_cache=/home/circleci/bazel_repository_cache
 # Limit Bazel to consuming 2560K of RAM
 build --local_resources=2560,1.0,1.0
 # Also limit Bazel's own JVM heap to stay within our 4G container limit
-startup --host_jvm_args=-Xmx1g
+startup --host_jvm_args=-Xmx2g
```

Comment:
maybe add a comment about how you measured the right value

Reply:
done